README.md

```diff
@@ -41,6 +40,26 @@ pen & paper RPG game design".
 * Docker
 
 
+## User Interface
+
+The webpage contains short project description and two forms with instructions.
```

Comment:
Chyba niepotrzebnie wspominamy o opisie, to ma być chyba bardziej instrukcja dla użytkownika niż dokumentacja... Może podmienić na "After app's successful startup you can see a webpage containing two blank forms"? Nie musi być dokładnie tak, to tylko przykład/sugestia